src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -130,10 +131,13 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 	private lastMouseWheelEventTime = 0;
 	private isMouseOverTabs = false;
 
+	private visible: boolean = false;
```

Comment:
I think this property can always do a dynamic lookup on the `tabsModel`, no? If the `tabsModel.count === 0`, then the tab bar should not be visible. Or can there be a condition otherwise?